src/tables/ChangeSet.tsx

```diff
@@ -0,0 +1,167 @@
+import InfoIcon from '@mui/icons-material/Info';
+import {
+    Box,
+    IconButton,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableFooter,
+    TableHead,
+    TablePagination,
+    TableRow,
+    Tooltip,
+} from '@mui/material';
+import { MouseEvent, useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import useChangeSetStore, {
+    CaptureState,
+    EntityMetadata,
+} from '../stores/ChangeSetStore';
+
+const getCapturesSelector = (state: CaptureState) => state.captures;
+
+function ChangeSetTable() {
+    const [page, setPage] = useState(0);
+    const rowsPerPage = 10;
+
+    const handleChangePage = (
+        event: MouseEvent<HTMLButtonElement> | null,
+        newPage: number
+    ) => {
+        setPage(newPage);
+    };
+
+    const captureState = useChangeSetStore(getCapturesSelector);
+    const captures = Object.values(captureState);
+
+    const captureDetails: EntityMetadata[] = captures.map(
+        (capture) => capture.metadata
+    );
+
+    const emptyRows =
+        page > 0
+            ? Math.max(0, (1 + page) * rowsPerPage - captureDetails.length)
+            : 0;
+
+    const intl = useIntl();
+    const columns = [
+        {
+            field: 'entityType',
+            headerIntlKey: 'changeSet.data.entityType',
+        },
+        {
+            field: 'name',
+            headerIntlKey: 'changeSet.data.entity',
+        },
+        {
+            field: 'user',
+            headerIntlKey: 'changeSet.data.user',
+        },
+        {
+            field: 'changeType',
+            headerIntlKey: 'changeSet.data.details',
+        },
+    ];
+
+    return (
+        <Box sx={{ mb: 2, mx: 2 }}>
+            {captures.length > 0 ? (
+                <TableContainer component={Box}>
```

Comment:
This applies to this entire table....

I switched back to the datagrid tables on other pages. I think we need to make a call on what we're gonna do for tables for the alpha. Kind of leaning towards just sticking with datagrid... but that kind of limits our abilities to add features to the table.

Reason I raise this in this PR is this is a clean implementation of a table and we could just make a component out or this. Let's chat about it more in a video chat.